src/pkg/count/keys.go

```diff
@@ -66,6 +66,12 @@ const (
 	StreamItemsErrored            Key = "stream-items-errored"
 	StreamItemsFound              Key = "stream-items-found"
 	StreamItemsRemoved            Key = "stream-items-removed"
+	Tombstones                    Key = "tombstones"
```

Comment:
removed tombstones, since it's not currently getting used.  I'll re-add it in the post-processing step.

In general, `TotalFooProcessed` is the count of "the number of times we saw this thing, strictly increasing even across resets and multiple deltas and etc; it's not a count of results, it's a count of the times we processed a Foo".